app/views/early_bird_dashboard/index.html.slim

```diff
@@ -62,3 +62,5 @@
                 br
               - else
                 span.constituency #{question.member_constituency}
+            - if feature_propose_ao_enabled?
+              p = link_to('Email Parli-branch about this question', "mailto:#{Settings.mail_reply_to}?subject=Question #{question.uin}")
```

Comment:
It's very minor, but you could use double quotes for both arguments.

Reply:
Thanks, good spot - we'll do that in the next PR 🙏